test/featuregates_linter/test_data/versioned_feature_list.yaml

```diff
@@ -978,10 +978,10 @@
     version: "1.32"
 - name: PodLogsQuerySplitStreams
   versionedSpecs:
-    - default: false
-      lockToDefault: false
-      preRelease: Alpha
-      version: "1.32"
+  - default: false
```

Comment:
If this is the result of running `hack/update-featuregates.sh`, why was this not caught by https://github.com/kubernetes/kubernetes/blob/master/hack/verify-featuregates.sh when the incorrectly indented lines were added?

Let's investigate and fix that together with fixing the file.

/cc @siyuanfoundation

Reply:
@pohly @siyuanfoundation Fixed in https://github.com/kubernetes/kubernetes/pull/129376/commits/3ba7a969de071885afeacec20159d126ba391f55

Run patched `hack/verify-featuregates.sh` on the master branch

```
(base) ➜  kubernetes git:(master) hack/verify-featuregates.sh
found 3 features in FeatureSpecMap of func featureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/component-base/logs/api/v1/kube_features.go
found 169 features in FeatureSpecMap var defaultVersionedKubernetesFeatureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/pkg/features/versioned_kube_features.go
found 2 features in FeatureSpecMap var defaultVersionedKubernetesFeatureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go
found 36 features in FeatureSpecMap var defaultVersionedKubernetesFeatureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/features/kube_features.go
found 1 features in FeatureSpecMap of func featureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/component-base/metrics/features/kube_features.go
found 2 features in FeatureSpecMap of func featureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/component-base/zpages/features/kube_features.go
found 1 features in FeatureSpecMap var versionedCloudPublicFeatureGates in file: /Users/kiki/workspace/golang/src/k8s.io/kubernetes/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go
Failed to verify versioned feature list:
detected diff in versioned feature list (test/featuregates_linter/test_data/versioned_feature_list.yaml), diff:
  bytes.Join({
  	... // 21460 identical bytes
  	": false\n    lockToDefault: false\n    preRelease: Alpha\n    versi",
  	"on: \"1.32\"\n- name: PodLogsQuerySplitStreams\n  versionedSpecs:\n  ",
- 	"- default: false\n",
+ 	"  - default: false\n  ",
  	"    lockToDefault: false\n",
+ 	"  ",
  	"    preRelease: Alpha\n",
+ 	"  ",
  	"    version: \"1.32\"\n- name: PodReadyToStartContainersCondition\n ",
  	" versionedSpecs:\n  - default: false\n    lockToDefault: false\n   ",
  	... // 12856 identical bytes
  }, "")
exit status 1
Please run 'hack/update-featuregates.sh' to update the feature list.
```

Reply:
Thanks @carlory!

/lgtm
/approve